pages/getting_started.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: page
+title: Getting Started 
+order: 1
+---
+
+MDAnalysis is a powerful Python library for analyzing molecular dynamics simulations. This page will guide you through quickly installing MDAnalysis, exploring a basic example, and accessing learning resources to get started.
+
+## Installation Instructions
+
+For a quick start, check out our [Installation Quick Start][] guide. 
+For a more thorough installation with advanced options, including dependencies and environment configurations, see the [installation instructions in the User Guide]({{ site.docs.userguide.url }}/stable/installation.html). 
+
+Once installed, explore this [basic example][] to load a trajectory and perform a simple analysis.
+
+[Installation Quick Start]: https://www.mdanalysis.org/pages/installation_quick_start/
```

Comment:
Do not use links with full protocol and hostoname inside the site. For this to work in local builds and on GitHub, use something like `{{ site.baseurl }}/pages/installation_quick_start/` ... but check how we are doing it elsewhere